Gemfile

```diff
@@ -4,3 +4,4 @@ gem 'nori'
 gem 'nokogiri'
 gem 'nokogiri-pretty'
 gem 'awesome_print'
+gem 'mini_portile'
```

Comment:
This maybe a dependency of nokogiri, but I am not using it directly so it shouldn't be in the Gemfile as a direct dependency.